packages/core/.mocharc.js

```diff
@@ -0,0 +1,15 @@
+// some tests in this package are run using mocha, not hardhat. This file a defines the entry point for these tests.
+
+// exit test runner on unhandled rejections
+process.on("unhandledRejection", (reason, promise) => {
+  console.error("Unhandled Rejection during test execution:", promise, "reason:", reason);
+  process.exit(1);
+});
+
+module.exports = {
+  require: ["ts-node/register/transpile-only"],
+  extension: ["js"],
+  watchExtensions: ["js"],
+  spec: ["test/insured-bridge/e2e/e2e_InsuredBridge.js"],
```

Comment:
Similar to the above, instead of specifying tests individually, maybe we could have a standard way of recognizing them so we wouldn't need to update any of these configs when we add new integration tests of this type.